main.go

```diff
@@ -60,8 +62,15 @@ func main() {
 	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
+	opts := zap.Options{
+		Development: true,
+	}
+	opts.BindFlags(flag.CommandLine)
+
 	flag.Parse()
 
+	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
```

Comment:
Why are we using `zap` instead of `klog` as agreed?

Reply:
```suggestion
	ctrl.SetLogger(klog.NewKlogr())
```

Reply:
Right fixed